schemas/snuba-queries.v1.schema.json

```diff
@@ -127,7 +127,33 @@
           "type": ["integer", "null"]
         },
         "stats": {
-          "type": "object"
+          "type": "object",
+          "properties": {
+            "final": {
+              "type": "boolean"
+            },
+            "cache_hit": {
```

Comment:
Does this mean the field is required? because I'm looking at the querylog data and it's not guaranteed to be there

Reply:
No, default is not required unless it's explicitly added to "required"